crates/api_crud/src/post/remove.rs

```diff
@@ -28,9 +29,10 @@ pub async fn remove_post(
   let post_id = data.post_id;
   let orig_post = Post::read(&mut context.pool(), post_id).await?;
 
+  let community = Community::read(&mut context.pool(), orig_post.community_id).await?;
```

Comment:
Read the post_view to get both in one query.

Reply:
Done.